conf/vbs.conf.part

```diff
@@ -129,7 +129,7 @@ source src_ch_vbs_retablierungsstellen : def_searchable_features
 
 source src_ch_vbs_schiessanzeigen : def_searchable_features
 {
-    sql_db = vbs_dev
+    sql_db = vbs_${DBSTAGING}
```

Comment:
One `_dev` left in line 151 that hasn't been replaced by `${DBSTAGING}`. Intentionally, or has it been hiding? :wink:

Reply:
done